src/lib/backend.ai-client-node.ts

```diff
@@ -223,7 +223,7 @@ class Client {
     this._features = {}; // feature support list
     this.abortController = new AbortController();
     this.abortSignal = this.abortController.signal;
-    this.requestTimeout = 5000;
+    this.requestTimeout = 10000;
```

Comment:
You should never touch this request timeout. Instead, please change timeout for session creation routine.